cmake/FindONNX.cmake

```diff
@@ -17,6 +17,14 @@ if(ONNXRT_ROOT_DIR)
 endif()
 
 if(ORT_LIB AND ORT_INCLUDE)
+  find_path(ORT_DML_INCLUDE dml_provider_factory.h
+    ${ONNXRT_ROOT_DIR}/include/onnxruntime/core/providers/dml
+    CMAKE_FIND_ROOT_PATH_BOTH)
+
+  if(ORT_DML_INCLUDE)
+    set(HAVE_ONNX_DML TRUE)
```

Comment:
if you will look for other headers/EPs as well, maybe it makes sense to make a macro

Reply:
Done